### ISSUE TITLE
User Story - Find jump scare content/ideas

### ISSUE THREAD

**Issue by KTC96** [KTC96/Didyouhearthat#1]
As a **developer** I want to add **jump scare content** to the page so that I can **create an authentic scary Halloween experience** for the user.

Acceptance Criteria
- Random jump scares appear during a user's interaction with the game.
- A variety of jump scares are available to keep the user engaged and scared.

Tasks
- [x] Find jump scares
- [x] Implement jump scares
- [x] Do not start videos until username is validated

**Comment by KTC96** [comment]
Found jump scares and added to project files, but they will not be implemented probably until saturday

**Comment by KTC96** [comment]
Added new task to make sure they only play after the player has inputted their username
